### ISSUE TITLE
Should anaconda projects enforce a `Projects Folder` and store this config?

### ISSUE THREAD

**Issue by goanpeca** [Anaconda-Platform/anaconda-project#47]
Pinging @havocp, @chdoig

Question, with Navigator we currently have an `Anaconda Projects` folder where we put all projects.  Is this a workflow we should also encourage with anaconda-project directly?

**Comment by havocp** [comment]
What do you have in mind specifically?

Maybe whenever we take a `--directory` option you could specify it relative to that folder? I think it may be almost a 99% case with `anaconda-project` that people do everything while cd'd into the project dir instead of using `--directory` though, so I don't know if that matters too much.

**Comment by chdoig** [comment]
I don't think this matters either. I think this is a Navigator configuration, not an Anaconda Project one. 
This is related to the discussion we had here: https://github.com/ContinuumIO/navigator/issues/1018, where users complained that we had created an AnacondaProject without their knowledge.
I'd not worry about it in AnacondaProject, you should be able to create projects wherever.

**Comment by mcg1969** [comment]
Thanks for opening this issue, and apologies for the long delay in responding.

Anaconda Project has not been actively developed in some time and has largely been superseded by newer alternatives:

* [pixi](https://pixi.sh/) and [uv](https://docs.astral.sh/uv/) represent the first truly successful successors.
* [conda-workspaces](https://github.com/conda-incubator/conda-workspaces) is a promising pixi-compatible alternative for users who prefer to stay closer to the conda ecosystem.

Going forward, work on this repository is focused on critical bug fixes and on tooling to help users migrate existing Anaconda Project projects to pixi (and, by extension, conda-workspaces). See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page in our documentation for details on the `anaconda-project export-pixi` command, which can convert an existing `anaconda-project.yml` into a `pixi.toml` for you.

The `anaconda-project` package itself will remain available, so existing projects relying on its stable feature set can continue to be supported. However, we do not anticipate further enhancement work, and for that reason we are closing out the bulk of our open issue backlog — including this one.

If this issue describes a critical regression in the stable feature set (as opposed to an enhancement or an edge case in a feature we no longer plan to expand) and you have a minimal reproducer against a current release, please feel free to reopen and we will take another look.

Thank you for your interest in Anaconda Project.
